api_app/script_analyzers/observable_analyzers/inquest.py

```diff
@@ -0,0 +1,62 @@
+import logging
+import requests
+
+from api_app.exceptions import AnalyzerRunException
+from api_app.script_analyzers.classes import ObservableAnalyzer
+from intel_owl import secrets
+
+
+logger = logging.getLogger(__name__)
+
+
+class InQuest(ObservableAnalyzer):
+    base_url: str = "https://labs.inquest.net"
+
+    def set_config(self, additional_config_params):
+        self.api_key_name = additional_config_params.get(
+            "api_key_name", "INQUEST_API_KEY"
+        )
+        self.__api_key = secrets.get_secret(self.api_key_name)
+        self.analysis_type = additional_config_params.get(
+            "inquest_analysis", "dfi_search"
+        )
+        self.hash_type = additional_config_params.get("hash_type", "md5")
+
+    def run(self):
+        result = {}
+        headers = {"Content-Type": "application/json"}
+        if self.__api_key:
+            headers["Authorization"] = self.__api_key
+        else:
+            warning = f"No API key retrieved with name: {self.api_key_name}"
+            logger.info(
+                f"{warning}. Continuing without API key..." f" <- {self.__repr__()}"
+            )
+            self.report["errors"].append(warning)
+
+        if self.analysis_type == "dfi_search":
+            if self.observable_classification == "hash":
+                uri = (
+                    f"/api/dfi/search/hash/{self.hash_type}?hash={self.observable_name}"
+                )
+            elif self.observable_classification in ["ip", "url", "domain"]:
+                uri = "/api/dfi/search/ioc/{0}?keyword={1}".format(
+                    self.observable_classification, self.observable_name
+                )
+            else:
+                raise AnalyzerRunException()
+
+        elif self.analysis_type == "iocdb_search":
+            uri = f"/api/iocdb/search?keyword={self.observable_name}"
+
+        elif self.analysis_type == "repdb_search":
```

Comment:
need an `else` condition here, that would also fix the LGTM error

Reply:
the error message in the `else` condition would be somewhat similar to [this](https://github.com/intelowlproject/IntelOwl/blob/1d1d61fd318c686776848d58733e6877d39b5ee1/api_app/script_analyzers/observable_analyzers/shodan.py#L31-L33)